PhysicsTools/HeppyCore/python/framework/eventsfwlite.py

```diff
@@ -15,16 +18,19 @@
 
 class Events(object):
     def __init__(self, files, tree_name,  options=None):
-	if options is not None :
-		if not hasattr(options,"inputFiles"):
-		 	options.inputFiles=files
-		if not hasattr(options,"maxEvents"):
-			options.maxEvents = 0	
-		if not hasattr(options,"secondaryInputFiles"):
-			options.secondaryInputFiles = []
-	        self.events = FWLiteEvents(options=options)
+        logging.info('opening input files:')
+        logging.info(pprint.pformat(files))
+        if options is not None :
+            if not hasattr(options,"inputFiles"):
+                options.inputFiles=files
+            if not hasattr(options,"maxEvents"):
+                options.maxEvents = 0	
+            if not hasattr(options,"secondaryInputFiles"):
+                options.secondaryInputFiles = []
```

Comment:
if you add logging, I'd suggest adding
````
elif options.secondaryInputFiles: # only if it's a non-empty list
    logging.info('using secondary input files: %s' % pprint.pformat(options.secondaryInputFiles))
````

Reply:
Done, but cannot really test it (never used this option)